common/components/enums/IssueArea.js

```diff
@@ -0,0 +1,22 @@
+const IssueArea = {
```

Comment:
Totally agree that these definitions should be deduplicated. That CSV file currently contains the definitions for several enumerated types, so looks like a refactor will need to be done before this can be safely used for just the Issue Area enum.

tabling for now and tracking in #13

Reply:
Oh, I see the Category column delineates which enum the values belong to! Will work on getting rid of this enum and replacing it with the service call you describe